### PR TITLE
Add support for initializing InflowWind with string inputs

### PR DESCRIPTION
**Pull request by rafmudaf**
**Feature or improvement description**
This pull request uses the recently added infrastructure to NWTC Library to parse string arrays as inputs. The objective is to allow driver codes to instantiate the InflowWind module without an input file.

**Related issue, if one exists**
None

**Impacted areas of the software**
InflowWind

**Test results, if applicable**
GH Actions - unit tests are included

### CODE REVIEW

File: modules/inflowwind/src/InflowWind_Subs.f90 [hunk 1]
```diff
@@ -120,192 +120,64 @@ MODULE InflowWind_Subs
...
-   END IF
-
-      ! If we are Echoing the input then we should re-read the first three lines so that we can echo them
-      ! using the NWTC_Library routines.  The echoing is done inside those routines via a global variable
-      ! which we must store, set, and then replace on error or completion.
-
-   IF ( InputFileData%EchoFlag ) THEN
-
-      CALL OpenEcho ( UnitEcho, TRIM(EchoFileName), TmpErrStat, TmpErrMsg )
-      CALL SetErrStat( TmpErrStat, TmpErrMsg, ErrStat, ErrMsg, RoutineName )
```
Comment (bjonkman) [inline modules/inflowwind/src/InflowWind_Subs.f90:237]: The new input file parsing here ignores this `EchoFlag`.

Comment (andrew-platt) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: Interesting.  I'll take a look at it.  Not sure what I broke there.

Comment (bjonkman) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: Don't change anything, yet, though. I'm trying to merge this (from dev) into Jason's FAST.Farm branch and have to fix some merge conflicts. I might change some of this while I am fixing those conflicts.

Comment (andrew-platt) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: Huh.  Looks like that is completely missing. I'll fix it.

Comment (bjonkman) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: Don't give me more merge conflicts!!!! :)

Comment (bjonkman) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: In my branch, I am now passing the `EchoFileName` to `InflowWind_ParseInputFileInfo`. I'll let you fix the parts missing the echo file inside the routine.

Comment (andrew-platt) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: Perfect.  Which branch?  I just setup a branch with a fix in that routine -- can't test it at the moment as I broke my compiler setup... :(
https://github.com/andrew-platt/openfast/commit/578d90c17e848da93bd3a76d7433069c838ee9d4

Comment (bjonkman) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: Thanks! I'm working on my f/fast-farm branch, but haven't pushed the changes you'll need, yet, as there are still several broken things with the InflowWind conflicts. I can grab your changes when I'm finished with this.

Comment (andrew-platt) [reply modules/inflowwind/src/InflowWind_Subs.f90:237]: Fixed a few typos in my branch.  Ended up passing the filename also (I find it useful to include that info in the echo file).

**Pull request merged**